pyvista/examples/downloads.py

```diff
@@ -4013,6 +4013,70 @@ def download_lucy(load=True):  # pragma: no cover
     return _download_and_read('lucy.ply', load=load)
 
 
+def download_pump_bracket(load=True):  # pragma: no cover
+    """Download the pump bracket example dataset.
+
+    Data generated from public SimScale examples at `SimScale Project Library -
+    Turbo <https://www.simscale.com/projects/STR/bracket/>`_.
+
+    Licensing for this dataset is granted to freely and without restriction
+    reproduce, distribute, publish according to the `SimScale Terms and
+    Conditions <https://www.simscale.com/terms-and-conditions/>`_.
```

Comment:
```suggestion
    """Download the pump bracket example dataset.

    Data generated from public SimScale examples at `SimScale Project Library -
    Turbo <https://www.simscale.com/projects/STR/bracket/>`_.

    Licensing for this dataset is granted freely and without restriction
    reproduce, distribute, and publish according to the `SimScale Terms and
    Conditions <https://www.simscale.com/terms-and-conditions/>`_.
```

Reply:
@tkoyama010 the "to" was correct where it was. The sentence could be parsed as
> Licensing for this dataset (is granted to [freely and without restriction] reproduce, distribute, publish) according to the SimScale Terms and Conditions

Without "to" it would start off as

> Licensing for this dataset (is granted freely and without restriction) (???) reproduce

Perhaps the "to" could be moved to

> licensing ... is granted freely and without restriction to reproduce, distibute, publish...

The question is whether the "freely and without restriction" related to the "granting" or the "reproducing".

@akaszynski, is this a quote from some pre-existing legalese document, or did you phrase it yourself?

Reply:
Ah. I got it thanks! English is difficult :)

Reply:
> @akaszynski, is this a quote from some pre-existing legalese document, or did you phrase it yourself?

It's my own words, and I agree it's a bit misworded. I've corrected it according to your suggestion in 45c56c2.